metashare/api/sf_org_changes.py

```diff
@@ -9,16 +9,26 @@
 from cumulusci.tasks.salesforce.sourcetracking import retrieve_components
 from django.conf import settings
 
-from .gh import get_repo_info, local_github_checkout
+from .gh import get_repo_info, get_source_format, local_github_checkout
 from .sf_run_flow import refresh_access_token
 
 
-def get_valid_target_directories(scratch_org_config):
+def get_valid_target_directories(user, scratch_org, repo_root):
     """
     Expects to be called from within a `local_github_checkout`.
     """
     package_directories = {}
-    if scratch_org_config.get("source_format") == "sfdx":
```

Comment:
This was not working, because `source_format` is not stored on the `scratch_org_config` -- it's part of the _package_ config from cumulusci.yml (like the branch prefix).